packages/docs/src/examples/data-tables/slots.vue

```diff
@@ -52,12 +52,14 @@
 
       <template v-if="isEnabled('body')" v-slot:body="{ items }">
         <tr v-for="item in items" :key="item.name">
-          <td>{{ item.name }}</td>
-          <td>CONTENT</td>
-          <td>CONTENT</td>
-          <td>CONTENT</td>
-          <td>CONTENT</td>
-          <td>CONTENT</td>
+          <tbody>
```

Comment:
`tr > tbody > td` doesn't look like a correct markup

Reply:
Sorry, now it's correct